src/minutes/2019-07-24.md

```diff
@@ -8,19 +8,107 @@ List the attendees for the meeting
 
 ## Agenda
 
+Please add new items to the meeting Agenda
+
+### Scipy Sprints
+* [x] Update from conda-forge Sprint (Filipe / Marius / Lori)
+    * Mugs next year?
+    * No one really wanted to work on infrastructure.
+    * Turned into more of a helpdesk -- Still a lot of issues out there. Mostly around documentation.
+        * How do I handle strict?
+* [x] Migration of Robot Operating System (ROS?) packaging ecosystem to conda-forge? Wolf Volprecht -- Who talked to him about this? I just overheard it. Maybe that was Anthony / Michael?
+    * Wolf is interested in adding these packages to conda-forge. Will probably be handled the same way as "r", so put a "ros-" prefix on the packages. Marius to reach out.
+* [x] Pypy implementation proposal: https://docs.google.com/document/d/1DdOEwmG3pr4m9kIktc1crp3dJPdvJjbPvidmEclpuCA/edit
+    * We have the CI bandwidth to build up the pypy ecosystem -- that's not a particularly painful thing to do. It's mostly around getting these things to even build. pypy has internal versions (6, 7, 8, etc.) that have compatibilities with specific versions of CPython, but the versioning scheme is different.
+    * Q: How will the pypy variant interact with other variants? It'll be another top-level variant that has to be mutually exclusive with the CPython variant.
+    * pypy has a different directory structure -- there may need to be changes on the conda side to support `noarch: pypy`.
+    * Next steps: Say yes / no to the spec, point out potential pitfalls, give it back to (Matti?) and say here's the work that needs to be done.
+* Marius did a vague hack at scipy to get stuff building on k8s on Azure. We can probably build anything that is Linux-like for things that take too long for CI. Things TODO: Grab the artifacts and upload them to anaconda.org. This is likely in a state that someone else can pick up and run with.
```

Comment:
@mariusvniekerk This sounds interesting but quite similar to what Gitlab offers. Where can I get more information about this?

Reply:
I have some initial sketch of the implementation here

https://github.com/mariusvniekerk/k8s-builder